views/js/apipCreator/test/helper/parser.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=EDGE" />
+        <title>APIP XML file parser</title>
+        <link rel="stylesheet" type="text/css" href="/tao/views/js/lib/qunit/qunit.css">
+        <script type="text/javascript" src="/tao/views/js/lib/require.js"></script>
+        <script type="text/javascript" src="/tao/views/js/lib/qunit/qunit.js"></script>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+        <script  type="text/javascript">
+            require(
+                [
+                    '/taoQtiItem/views/js/apipCreator/helper/parser.js',
+                    '/taoQtiItem/views/js/apipCreator/helper/serializer.js',
+                ], 
+                function (parser, serializer) {
+                $.get(
+                    '/taoQtiItem/views/js/apipCreator/test/assets/apip_example_exemplar01.xml'
```

Comment:
you would have loaded the sample xml with requirejs, this would make your test clearer.
require('text!taoQtiItem/views/js/apipCreator/test/assets/apip_example_exemplar01.xml', function(xmlString){})